### PR TITLE
reduce cpu-usage (#62)

### PR DESCRIPTION
**Pull request by hersche**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -48,8 +48,8 @@
     "player"
   ],
   "dependencies": {
-    "video.js": ">=6.0.0",
-    "wavesurfer.js": ">=2.0.3"
+    "video.js": ">=6.8.0",
+    "wavesurfer.js": ">=2.0.5"
   },
```
Comment (thijstriemstra) [inline package.json]: @hersche can you revert these changes? we don't depend on any newer version.

Comment (hersche) [reply package.json]: shure, but i also tracked another issue in the testing-class..
you can also do a alternative solution to the counter, it's just workarounded so i can use it

**Comment by hersche** [comment]
mh, i don't get it for the moment. if you have a clear solution in mind (instead of the ugly counter), just ignore the PR and i will simply update from you.. that's propably more easy..

**Comment by thijstriemstra** [comment]
Thanks for the pull request @hersche but it looks like this workaround isn't needed when disabling the custom tech (see notes in #62). Closing.

**Comment by thijstriemstra** [comment]
Reopening as discussed in https://github.com/collab-project/videojs-wavesurfer/issues/62#issuecomment-391184197

**Comment by thijstriemstra** [comment]
thanks, that looks good. Only you might as well remove the counter var, it doesn't seem to do anything right now.

**Comment by hersche** [comment]
yeah, i just don't know how to solve the coveralls..

**Comment by thijstriemstra** [comment]
> yeah, i just don't know how to solve the coveralls..

I think by deleting the actual `src/tech.js` as well.

**Comment by thijstriemstra** [comment]
There are still references to `this.progressCounter` and the commented out code can be removed as well (inc any of its comments).

File: src/js/videojs.wavesurfer.js [hunk 2]
```diff
@@ -574,6 +569,7 @@ class Wavesurfer extends Plugin {
      */
     onWaveFinish() {
         this.log('Finished playback');
+        this.setCurrentTime();
```
Comment (thijstriemstra) [inline src/js/videojs.wavesurfer.js]: unnecessary call.

File: src/js/videojs.wavesurfer.js [hunk 3]
```diff
@@ -630,7 +626,6 @@ class Wavesurfer extends Plugin {
     onWaveError(error) {
         // notify listeners
         this.player.trigger('error', error);
-
```
Comment (thijstriemstra) [inline src/js/videojs.wavesurfer.js]: this whitespace was fine imo.

File: src/js/videojs.wavesurfer.js (hunk 2 above)
Comment (hersche) [reply src/js/videojs.wavesurfer.js]: withouth this, it showed me a incorrect time, when the track has ended. even withouth the counter.

**Comment by hersche** [comment]
i think it's reverted as you whished, i do not need it like this and i'm not shure if something happen here, but i always see it in my dashboard, so i feel so free and close it unmerged.
